starknet_py/abi/v1/abi.ebnf

```diff
@@ -0,0 +1,21 @@
+IDENTIFIER: /[a-zA-Z_][a-zA-Z_0-9]*/
+
+start: type
+     | struct
+
+type: "core" "::" core_value
+    | type_unit
+
+core_value: "felt" "252"                                        -> type_felt
+          | "integer" "::" "u" INT                              -> type_uint
+          | "option" "::" "Option" "::" "<" (type | struct) ">" -> type_option
+          | "array" "::" "Array" "::" "<" (type | struct) ">"   -> type_array
+
+type_unit: "()"
+
+struct: (IDENTIFIER "::")+ "<" type ">"
```

Comment:
Not sure abut this, why is struct defined like that? 

Can't we have a method that accept for example just this `account::account::Call`? How would that be matched?

Reply:
It won't. I'm adding this based on the examples, so once structs will be included correctly (I believe it just happened) I'll modify them